src/provider/connectors.ts

```diff
@@ -23,7 +23,7 @@ export const CHAIN_ID_BY_NAME = {
   'develop': 66,
 };
 
-export const DEFAULT_ETH_NETWORK = ETH_NETWORKS[0];
```

Comment:
this should be index 0. When using yarn dev the networks are 'develop,mainnet,kovan' and in yarn start we are suing 'mainnet,kovan'

Also changing this line shouldn't affect the rest of this changes, since you get the chainID from activeWeb3.

Reply:
fixed